python/lsst/dax/distribution/DataGenClient.py

```diff
@@ -63,7 +63,8 @@ def __init__(self, host, port, target_dir='fakeData', chunks_per_req=5):
         self._cl_conn = None # DataGenConnection
         self._cfg_file_name = 'gencfg.py' # name of the local config file for the generator
         self._cfg_file_contents = None # contents of the config file.
-        self._datagenpy = '~/work/dax_data_generator/bin/datagen.py' # TODO: MUST stop hard coding this
+        #self._datagenpy = '~/work/dax_data_generator/bin/datagen.py' # TODO: MUST stop hard coding this &&&
+        self._datagenpy = '~/dax_data_generator/bin/datagen.py' # TODO: MUST stop hard coding this &&&
```

Comment:
This is very unsafe. There are two reasons for this First of all, `~` implies the current user which may potentially be different from a location where you put the package. The second problem is that your package may be installed at some other location.
Assuming that the current script is imported into one of your binary tools deployed at the package's `bin/` folder you may try to figure out an absolute path of that folder by:
```
import os.path
import sys
bindir = os.path.dirname(os.path.abspath(sys.argv[0]))
if bindir == '': basedir = '.'
```
After that you could do:
```
self._datagenpy = '{$s}/datagen.py'.format(bindir)
```

Reply:
The other ticket I'm working on is largely about removing this bit code and calling DataGenerator in python without calling an outside procedure. That is done and this hard coded line is gone in the other ticket.

Reply:
👍